rules/windows/malware/win_mal_ryuk.yml

```diff
@@ -11,10 +11,13 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine:
-            - '*\net.exe stop "samss" *'
-            - '*\net.exe stop "audioendpointbuilder" *'
-            - '*\net.exe stop "unistoresvc_?????" *'
+        CommandLine|contains|all:
+            - 'net.exe'
+            - 'stop'
+        CommandLine|contains:
```

Comment:
it's better to move the `net.exe` to ProcessName field, cheaper from computing resources perspective  (:

Reply:
Done :)